scripts/format_code.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
this is helpful to keep things clean, previous the style was a bit heterogeneous

Reply:
Is this just touching the CMake files right now and not the source code files?

Reply:
Correct